src/platform/Linux/MdnsImpl.cpp

```diff
@@ -79,6 +81,29 @@ chip::Inet::IPAddressType ToAddressType(AvahiProtocol protocol)
     return type;
 }
 
+#if 0
```

Comment:
should we delete commented-out code? or is this intended somehow (if so ,  please add a comment)

Reply:
Yes, sorry, I forgot I kept the old code while testing.